pkg/config/v1alpha1/types.go

```diff
@@ -134,6 +138,17 @@ type ControllerHealth struct {
 	LivenessEndpointName string `json:"livenessEndpointName,omitempty"`
 }
 
+// ControllerProfiling defines profiling configuration for controllers.
+type ControllerProfiling struct {
+	// PprofBindAddress is the TCP address that the controller should bind to
+	// for serving pprof.
+	// It can be set to "" or "0" to disable the pprof serving.
+	// Since pprof may contain sensitive information, make sure to protect it
+	// before exposing it to public.
+	// +optional
+	PprofBindAddress string `json:"pprofBindAddress,omitempty"`
+}
```

Comment:
Given the component configuration types got deprecated, can we remove these?

Reply:
done